amazon-msk/cdk/secure-private-access/README.md

```diff
@@ -0,0 +1,390 @@
+            }
+          ]
+        }
+      }
+    ]
+  }'
+```
+
+
+### Launch Client EC2 Instance in different VPC and Install the Kafka Client
+
+Follow the AWS guide to launch an EC2 instance to be able to connect to MSK Serverless.
+https://docs.aws.amazon.com/msk/latest/developerguide/create-serverless-cluster-client.html
+
+This documentation mentions that "under VPC, enter the ID of the virtual private cloud (VPC) for your serverless cluster". Now that you deployed Zilla Plus, you can provide the client VPC where you configured the VPC Endpoint. 
+
+Make sure to download one of the latest versions of `aws-msk-iam-auth` jar file if you want to use `OATHBEARER` authentication mechanism, as that's not included in `v1.1.1`
+
+For example:
+```bash
+wget https://github.com/aws/aws-msk-iam-auth/releases/download/v2.2.0/aws-msk-iam-auth-2.2.0-all.jar
+```
+
+
+### Configure the Kafka Client
+
+With the Kaka client now installed we are ready to configure it and point it at the custom domain configured for Zilla Plus.
+
+You can either choose to use `AWS_MSK_IAM` or `OAUTHBEARER`.
+
+##### client.properties for `AWS_MSK_IAM`
+
+```text
+security.protocol=SASL_SSL
+sasl.mechanism=AWS_MSK_IAM
+sasl.jaas.config=software.amazon.msk.auth.iam.IAMLoginModule required;
+sasl.client.callback.handler.class=software.amazon.msk.auth.iam.IAMClientCallbackHandler
+```
+
+##### client.properties for `OAUTHBEARER`
+
+```text
+security.protocol=SASL_SSL
+sasl.mechanism=OAUTHBEARER
+sasl.jaas.config=org.apache.kafka.common.security.oauthbearer.OAuthBearerLoginModule required;
+sasl.login.callback.handler.class=software.amazon.msk.auth.iam.IAMOAuthBearerLoginCallbackHandler
+sasl.client.callback.handler.class=software.amazon.msk.auth.iam.IAMOAuthBearerLoginCallbackHandler
+```
+
+TODO:
+- add iam role readme
+- add steps to add more supported regions, point to the filed issue that limits us to automate this
+
+
+### Test the Kafka Client
+
+This verifies internet connectivity to your MSK Serverless via Zilla Plus.
+
+We can now verify that the Kafka client can successfully communicate with your MSK Serverless from your EC2 instance running in a different VPC to create a topic, then publish and subscribe to the same topic.
+
+If using the wildcard DNS pattern `*.example.aklivity.io`, then we use the following server name for the Kafka client:
+
+```text
+boot.example.aklivity.io:9098
+```
+
+Replace these bootstrap server names accordingly for your own custom wildcard DNS pattern.
+
+#### Create a Topic
+
+Use the Kafka client to create a topic called zilla-proxy-test, replacing <tls-bootstrap-server-names> in the command below with the TLS proxy names of your Zilla proxy:
+
+```bash
+bin/kafka-topics.sh --create --topic zilla-proxy-test --partitions 3 --replication-factor 2 --command-config client.properties --bootstrap-server <tls-bootstrap-server-names>
```

Comment:
```suggestion
bin/kafka-topics.sh --create \
    --topic zilla-proxy-test \
    --partitions 3 \
    --replication-factor 2 \
    --command-config client.properties \
    --bootstrap-server boot.us-east-1.example.aklivity.io:9098
```

Use the example, and change the instruction to replace `boot.us-east-1.example.aklivity.io:9098` instead.